Dockerfile.invocation-image

```diff
@@ -0,0 +1,23 @@
+ARG ALPINE_VERSION=3.8
+ARG GO_VERSION=1.11.0
+
+FROM golang:${GO_VERSION}-alpine${ALPINE_VERSION} AS build
+ARG DOCKERCLI_VERSION=18.03.1-ce
+ARG DOCKERCLI_CHANNEL=edge
+RUN apk add --no-cache \
+  make
+
+WORKDIR /go/src/github.com/docker/app/
+
+# builder of invocation image entrypoint
+FROM build AS invocation-build
```

Comment:
Minor point, but given the simplicity and special purposeness of this Dockerfile perhaps the distinction between these `build` and `build-invocation` stages is not useful?

Reply:
I can merge them yes, I wanted to make this one and the `Dockerfile` to resemble as much as possible.